src/index.html

```diff
@@ -7,6 +7,41 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main-page">
+      <div class="calendar calendar--start-day_sun calendar--month-length_29">
```

Comment:
From the checklist:

- BEM - Make sure to follow BEM naming convention for complex modifiers: `block-name--modifier-name--modifier-value`

Let's remove that underscore and use a dash ;)

Reply:
On video lessons we were learned to use values of modifiers with underscore:
block-name__element-name--modifier-name_modifier-value
so in the case **calendar--start-day_sun** and **calendar--month-length_29** are OK.
Please check BEM video: https://prnt.sc/fY_2AOWUhfLb